projects/core/src/anonymous-consents/store/effects/anonymous-consents.effect.ts

```diff
@@ -14,6 +21,8 @@ export class AnonymousConsentsEffects {
     AnonymousConsentsActions.LoadAnonymousConsentTemplates
   > = this.actions$.pipe(
     ofType(SiteContextActions.LANGUAGE_CHANGE),
+    withLatestFrom(this.authService.isUserLoggedIn()),
+    filter(([_, isUserLoggedIn]) => !isUserLoggedIn),
```

Comment:
Don't know where to put this, but as discussed, we will update the store appropriately for anom consent when the user is logged in. (reminder)